### PR TITLE
Deactivate tables auditing during migrations

### PR DESCRIPTION
**Pull request by elemoine**
Fixes https://github.com/qwat/QWAT/issues/248.

Depends on #232, which should be merged first.

### CODE REVIEW

**Comment by elemoine** [comment]
Now rebased and ready for review.

File: update/delta/pre-all.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+DO $$
+BEGIN
+
+    -- before 1.2.3 deactivate_audit_tables did not exist, so we test that the function
```
Comment (lbartoletti) [inline update/delta/pre-all.sql]: 1.3.2?

Comment (elemoine) [reply update/delta/pre-all.sql]: Good catch! Fixed.

**Comment by haubourg** [comment]
Merci !

**Pull request merged**